src/rlx_config.erl

```diff
@@ -225,20 +225,20 @@ get_patch_count(RawRef) ->
 build_vsn_string(Vsn, RawRef, RawCount) ->
     %% Cleanup the tag and the Ref information. Basically leading 'v's and
     %% whitespace needs to go away.
-    RefTag = [".ref", re:replace(RawRef, "\\s", "", [global])],
-    Count = re:replace(RawCount, "\\s", "", [global]),
+    RefTag = [".ref", re:replace(RawRef, "\\W", "", [global])],
+    Count = re:replace(RawCount, "\\D", "", [global]),
```

Comment:
this sounds like it can vastly change semantics by dropping accent characters, non-english terms and so on. I.e. someone having a tag like "release 3.6" now gets "36" as a reftag instead of "release3.6". Also without unicode support passed in as an option you might be dropping even more than expected.

The count thing seems alright for what you get there.

Reply:
Good point about the W, I did not think about it, I'll revert that change